### PR TITLE
chore: Add holesky support configuration

### PR DESCRIPTION
**Pull request by Rekard0**
## Description

Add holesky support configuration

Task ID: [OS-1411](https://aragonassociation.atlassian.net/browse/OS-1411)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I created tasks to update dependent repositories (OSx, Plugins)
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

[OS-1411]: https://aragonassociation.atlassian.net/browse/OS-1411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: configs/src/deployments/json/holesky.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    "DAOENSSubdomainRegistrarImplementation": {
+      "address": "0x6d4FB6Ff01A172774f42789fcfcdd84E68c28494",
+      "blockNumber": 1842632,
+      "deploymentTx": "0x8c489dd5f838070daad7ba3691240b46b341270b17f5287d4b87028200b8a17f"
+    },
+    "ENSRegistry": {
+      "address": "0x6d4FB6Ff01A172774f42789fcfcdd84E68c28494",
+      "blockNumber": 1842632,
+      "deploymentTx": "0x8c489dd5f838070daad7ba3691240b46b341270b17f5287d4b87028200b8a17f"
+    },
```
Comment (clauBv23) [inline configs/src/deployments/json/holesky.json:52]: this is the Registrar address

Comment (clauBv23) [reply configs/src/deployments/json/holesky.json:52]: Holesky  supports ENS so this address is not needed since we are not deploying our own

Comment (Rekard0) [reply configs/src/deployments/json/holesky.json]: addressed at https://github.com/aragon/osx-commons/pull/92/commits/2850c12361340a1e0f5e186801ff58b1a58228c4

**Review by clauBv23 — APPROVED** [review]
LGTM!

**Pull request merged**